src/components/cubes.jsx

```diff
@@ -42,6 +48,22 @@ export function Cubes({ app, closeOnClickOutside }) {
     forceUpdate();
   }
 
+  useEffect(() => {
+    // Check if cubes are stored in localstorage.
+    let storedCubes = localStorage.getItem(app.id);
```

Comment:
I believe this can throw an error if local storage is disabled, we probably need to gracefully disable this feature if we cannot use LS.

Reply:
Good catch.  I added a check if localstorage is disabled.